app/components/bali/tabs/index.scss

```
@@ -1,4 +1,4 @@
-.tabs {
+.tabs-component {
   & > ul {
```

Creo que no es necesario que indiques que el ul es decendiente de `.tabs-component`
```suggestion
  ul {
```